plugin/src/sbt-test/scalafmt-sbt/sbt/test

```diff
@@ -98,13 +100,14 @@ $ copy-file p14/src/test/scala/Test3.scala p14/src/main/scala/Test3.scala
 > p14/scalafmtCheck
 > p14/scalafmt
 ######## cache invalidation expected when a file was updated
-$ touch p14/src/main/scala/Test.scala
+> updateLastModified p14/src/main/scala/Test.scala
 -> p14/scalafmtCheck
 
 > p15/scalafmt
 > p15/scalafmtCheck
 # prevent reading the source (without changing the mtime) to detect actual/uncached invocation of scalafmt
 $ exec chmod 000 p15/src/main/scala/Test.scala
+> set p15/Compile/unmanagedSources += file("p15/src/main/scala/Test.scala")
```

Comment:
add explicit because latest sbt filter `chmod 000` files.

```
$ tree src 
src
└── main
    └── scala
        ├── A.scala
        └── B.scala
```

```
example > show Compile/unmanagedSources
[info] * /Users/kenji/example/src/main/scala/B.scala
[info] * /Users/kenji/example/src/main/scala/A.scala
[success] Total time: 0 s, completed Sep 3, 2025, 10:06:25 PM
example > eval sys.process.Process(Seq("chmod", "000", "src/main/scala/B.scala")).!
[info] ans: Int = 0
example > show Compile/unmanagedSources
[info] * /Users/kenji/example/src/main/scala/A.scala
[success] Total time: 0 s, completed Sep 3, 2025, 10:07:17 PM
```